function-controller/pkg/controller/deployer.go

```diff
@@ -121,12 +122,18 @@ func (d *deployer) buildSidecarContainer(function *v1.Function) corev1.Container
 	if outputDestination == "" {
 		outputDestination = "replies"
 	}
+	initialDelayVal := "0"
+	if function.Spec.InitialDelayMs != nil {
```

Comment:
Sorry, I was wrong in the call. If the Function went through settings defaults, `0` should already be there.
Better not to encode the logic of that default in several places...